chat.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express();
+const http = require("http");
+const server = http.createServer(app);
+//const socket = require("socket.io");
+//const io = socket(server);
+const io = require("socket.io")(server, {
+  cors: { origin: "http://localhost:3000" },
+});
+const port = 4000;
```

Comment:
Two servers running simultaneously?

Reply:
Yeah, one runs with deno, the other with node